applications/index.adoc

```diff
@@ -28,14 +28,15 @@ endif::openshift-rosa,openshift-dedicated[]
 [id="creating-application"]
 === Creating an application
 
-To create applications, you must have created a project or have access to a project with the appropriate roles and permissions. You can create an application by using either xref:../applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc#odc-creating-applications-using-developer-perspective[the Developer perspective in the web console], xref:../applications/creating_applications/creating-apps-from-installed-operators.adoc#creating-apps-from-installed-operators[installed Operators], or xref:../applications/creating_applications/creating-applications-using-cli.adoc#creating-applications-using-cli[the {product-title} CLI]. You can source the applications to be added to the project from Git, JAR files, devfiles, or the developer catalog.
+To create applications, you must have created a project or have access to a project with the appropriate roles and permissions. You can create an application by using either xref:../applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc#odc-creating-applications-using-developer-perspective[the Developer perspective in the web console], xref:../applications/creating_applications/creating-apps-from-installed-operators.adoc#creating-apps-from-installed-operators[installed Operators], or xref:../applications/creating_applications/creating-applications-using-cli.adoc#creating-applications-using-cli[the OpenShift CLI (`oc`)]. You can source the applications to be added to the project from Git, JAR files, devfiles, or the developer catalog.
```

Comment:
Would it make sense to use the `{oc-first}` attribute here rather than spell out "OpenShift CLI (`oc`)"?

Reply:
I wasn't aware of that one. Thanks!